cms/templates/js/access-editor.underscore

```diff
@@ -31,15 +31,15 @@
     <% } %>
     <h3 class="modal-section-title"><%- gettext('Use as a Prerequisite') %></h3>
     <div class="modal-section-content gating-is-prereq">
-        <ul class="list-fields list-input">
-            <li class="field field-checkbox checkbox-cosmetic">
+        <div class="list-fields list-input">
+            <div class="field-checkbox checkbox-cosmetic">
```

Comment:
Having the class "field" was what was causing the dialog to scroll up when you pressed the checkbox-- the input was for some reason very large (though not visible).

I removed this particular ul/li nesting, but this file still has ul/li nesting above. I think there are likely other a11y issues in this file due to the checkbox and select, so it doesn't seem worth trying to completely fix this template now.